make.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+run () {
+    echo "Running application..."
+    go run -tags sdl . $1
+}
+
+update () {
+    echo "Updating Carcassonne-Engine..."
+    go get -u github.com/YetAnotherSpieskowcy/Carcassonne-Engine@main
+    echo "Updating raylib-go..."
+    go get -v -u github.com/gen2brain/raylib-go/raylib
+}
+
+if [ $1 == "run" ]; then
```

Comment:
POSIX sh (which is the shell you listed in the shebang) requires using single equal sign for equality checks:
```suggestion
if [ $1 = "run" ]; then
```